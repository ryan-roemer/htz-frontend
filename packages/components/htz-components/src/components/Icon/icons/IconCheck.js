/* *************************************************************** *
 * THIS IS AN AUTO GENERATED FILE. PLEASE DO NOT EDIT IT DIRECTLY.
 *
 * If you want to change this Icon component , it is generated
 * from the `iconTamplate.js` file the parent directory.
 * *************************************************************** */
import React from 'react';
import { FelaComponent, } from 'react-fela';
import iconStyle from '../iconStyle';
import { iconPropTypes, iconDefaultProps, } from '../iconPropTypes';

const IconCheck = ({ size, fill, color, attrs, miscStyles, ...props }) => (
  <FelaComponent
    size={size}
    fill={fill}
    color={color}
    attrs={attrs}
    miscStyles={miscStyles}
    rule={iconStyle}
    render={({ className, }) => (
      <svg width="1em" height="1em" viewBox="0 0 725.7 725.7" className={className} {...props} {...attrs} >  <path fill="currentColor" d="M563.4 228.9L301.5 536.6l-139.2-153 45.6-41.7 75.4 82.8 7.9 8.7 9.2 10.1 8.9-10.4 7.6-8.9L516.6 189l46.8 39.9zm139.5 133.9c0 187.5-152.6 340.1-340.1 340.1S22.7 550.4 22.7 362.8c0-187.5 152.6-340.1 340.1-340.1 46 0 90.7 9.1 132.7 26.9 40.7 17.3 77.2 42 108.5 73.4 31.2 31.3 55.6 67.6 72.6 108 17.6 41.8 26.5 86.2 26.3 131.8zm-26.5.7c0-172.9-140.7-313.6-313.6-313.6S49.3 190.6 49.3 363.5 189.9 677 362.8 677c42.4 0 83.6-8.4 122.4-24.8 37.5-15.9 71.2-38.7 100-67.7 28.7-28.8 51.2-62.3 66.9-99.5 16.3-38.5 24.5-79.4 24.3-121.5z" /></svg>
    )}
  />
);

IconCheck.propTypes = iconPropTypes;
IconCheck.defaultProps = iconDefaultProps;

export default IconCheck;
