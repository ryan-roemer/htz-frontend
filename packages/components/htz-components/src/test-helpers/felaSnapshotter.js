import React from 'react';
import createRenderer from '../components/StyleProvider/createRenderer';
import StyleProvider from '../components/StyleProvider/StyleProvider';
import createFelaSnapshotter from './createFelaSnapshotter';

const testRenderer = createRenderer({ isRtl: true, });
const felaSnapshotter = createFelaSnapshotter(StyleProvider, testRenderer);

export default felaSnapshotter;
