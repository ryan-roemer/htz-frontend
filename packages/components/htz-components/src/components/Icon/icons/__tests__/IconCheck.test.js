
/* global window */
/* *************************************************************** *
 * THIS IS AN AUTO GENERATED FILE. PLEASE DO NOT EDIT IT DIRECTLY.
 *
 * If you want to change the styleguide example, it is generated
 * from the `embedsFileTemplate.js` file is this directory.
 * *************************************************************** */
import React from 'react';
import felaSnapshotter from '../../../../test-helpers/felaSnapshotter';
import IconCheck from '../IconCheck';

describe('<IconCheck />', () => {
  /* The error tests are left as documentation, but are commented out for the redability of the test results */
  // describe('Errors', () => {
  //   it('throw an error when the value passed to the "size" prop isn't a number', () => {
  //     expect(() => felaSnapshotter(<IconCheck size="2rem" />)).toThrow(
  //       /^An Icon's "size" prop may only be passed a "number"/
  //     );
  //   });
  //   it('throw an error when the value passed to the "color" prop isn't a named color (string)', () => {
  //     expect(() => felaSnapshotter(<IconCheck color="red" />)).toThrow(
  //       /is not a named-color/
  //     );
  //   });
  //   it('throw an error when the value passed to the "color" prop isn't a named color (array of strings)', () => {
  //     expect(() =>
  //       felaSnapshotter(<IconCheck color={[ 'primary', 'nope', ]} />)
  //     ).toThrow(/is not a named-color/);
  //   });
  //   it('throw an error when the value passed to the "fill" prop isn't a named color (string)', () => {
  //     expect(() => felaSnapshotter(<IconCheck fill="red" />)).toThrow(
  //       /is not a named-color/
  //     );
  //   });
  //   it('throw an error when the value passed to the "fill" prop isn't a named color (array of strings)', () => {
  //     expect(() =>
  //       felaSnapshotter(<IconCheck fill={[ 'primary', 'nope', ]} />)
  //     ).toThrow(/is not a named-color/);
  //   });
  // });
  describe('DOM element', () => {
    testCase('render correctly when no props are passed', <IconCheck />);
    testCase(
      'pass attributes defined in the "attrs" prop to the DOM element',
      <IconCheck attrs={{ 'aria-hidden': true, }} />
    );
    testCase('add "id" attribute to DOM element', <IconCheck id="hasId" />);
    testCase(
      'add "onClick" attribute to DOM element',
      <IconCheck id={e => console.log('a')} />
    );
  });

  describe('styles', () => {
    testCase("set icon's color", <IconCheck color={[ 'primary', 'base', ]} />);
    testCase(
      "set icon's color responsively",
      <IconCheck
        color={[
          { until: 'xl', value: [ 'primary', 'base', ], },
          { from: 'xl', value: [ 'bodyText', 'base', ], },
        ]}
      />
    );
    testCase("set icon's fill", <IconCheck fill={[ 'primary', 'base', ]} />);
    testCase(
      "set icon's fill responsively",
      <IconCheck
        fill={[
          { until: 'xl', value: [ 'primary', 'base', ], },
          { from: 'xl', value: [ 'bodyText', 'base', ], },
        ]}
      />
    );
    testCase("set icon's size", <IconCheck size={6} />);
    testCase(
      "set icon's size responsively",
      <IconCheck
        size={[ { until: 'xl', value: 6, }, { from: 'xl', value: 7, }, ]}
      />
    );
    testCase(
      'set misc styles on an icon',
      <IconCheck miscStyles={{ position: 'relative', }} />
    );
    testCase(
      'responsively set misc styles on an icon',
      <IconCheck
        miscStyles={{
          position: [
            { until: 'xl', value: 'relative', },
            { from: 'xl', value: 'static', },
          ],
        }}
      />
    );
  });
});

function testCase(name, Component, { only = false, } = {}) {
  const testFunc = only ? it.only : it;
  testFunc(name, () => {
    const { component, styles, } = felaSnapshotter(Component);
    expect(component).toMatchSnapshot();
    expect(styles).toMatchSnapshot();
  });
}
