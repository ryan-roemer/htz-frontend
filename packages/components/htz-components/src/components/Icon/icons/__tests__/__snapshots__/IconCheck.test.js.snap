// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconCheck /> DOM element add "id" attribute to DOM element 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c"
    height="1em"
    id="hasId"
    viewBox="0 0 725.7 725.7"
    width="1em"
  >
      
    <path
      d="M563.4 228.9L301.5 536.6l-139.2-153 45.6-41.7 75.4 82.8 7.9 8.7 9.2 10.1 8.9-10.4 7.6-8.9L516.6 189l46.8 39.9zm139.5 133.9c0 187.5-152.6 340.1-340.1 340.1S22.7 550.4 22.7 362.8c0-187.5 152.6-340.1 340.1-340.1 46 0 90.7 9.1 132.7 26.9 40.7 17.3 77.2 42 108.5 73.4 31.2 31.3 55.6 67.6 72.6 108 17.6 41.8 26.5 86.2 26.3 131.8zm-26.5.7c0-172.9-140.7-313.6-313.6-313.6S49.3 190.6 49.3 363.5 189.9 677 362.8 677c42.4 0 83.6-8.4 122.4-24.8 37.5-15.9 71.2-38.7 100-67.7 28.7-28.8 51.2-62.3 66.9-99.5 16.3-38.5 24.5-79.4 24.3-121.5z"
      fill="currentColor"
    />
  </svg>
</div>
`;

exports[`<IconCheck /> DOM element add "id" attribute to DOM element 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}"`;

exports[`<IconCheck /> DOM element add "onClick" attribute to DOM element 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c"
    height="1em"
    id={[Function]}
    viewBox="0 0 725.7 725.7"
    width="1em"
  >
      
    <path
      d="M563.4 228.9L301.5 536.6l-139.2-153 45.6-41.7 75.4 82.8 7.9 8.7 9.2 10.1 8.9-10.4 7.6-8.9L516.6 189l46.8 39.9zm139.5 133.9c0 187.5-152.6 340.1-340.1 340.1S22.7 550.4 22.7 362.8c0-187.5 152.6-340.1 340.1-340.1 46 0 90.7 9.1 132.7 26.9 40.7 17.3 77.2 42 108.5 73.4 31.2 31.3 55.6 67.6 72.6 108 17.6 41.8 26.5 86.2 26.3 131.8zm-26.5.7c0-172.9-140.7-313.6-313.6-313.6S49.3 190.6 49.3 363.5 189.9 677 362.8 677c42.4 0 83.6-8.4 122.4-24.8 37.5-15.9 71.2-38.7 100-67.7 28.7-28.8 51.2-62.3 66.9-99.5 16.3-38.5 24.5-79.4 24.3-121.5z"
      fill="currentColor"
    />
  </svg>
</div>
`;

exports[`<IconCheck /> DOM element add "onClick" attribute to DOM element 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}"`;

exports[`<IconCheck /> DOM element pass attributes defined in the "attrs" prop to the DOM element 1`] = `
<div>
  <svg
    aria-hidden={true}
    className="htz-a htz-b htz-c"
    height="1em"
    viewBox="0 0 725.7 725.7"
    width="1em"
  >
      
    <path
      d="M563.4 228.9L301.5 536.6l-139.2-153 45.6-41.7 75.4 82.8 7.9 8.7 9.2 10.1 8.9-10.4 7.6-8.9L516.6 189l46.8 39.9zm139.5 133.9c0 187.5-152.6 340.1-340.1 340.1S22.7 550.4 22.7 362.8c0-187.5 152.6-340.1 340.1-340.1 46 0 90.7 9.1 132.7 26.9 40.7 17.3 77.2 42 108.5 73.4 31.2 31.3 55.6 67.6 72.6 108 17.6 41.8 26.5 86.2 26.3 131.8zm-26.5.7c0-172.9-140.7-313.6-313.6-313.6S49.3 190.6 49.3 363.5 189.9 677 362.8 677c42.4 0 83.6-8.4 122.4-24.8 37.5-15.9 71.2-38.7 100-67.7 28.7-28.8 51.2-62.3 66.9-99.5 16.3-38.5 24.5-79.4 24.3-121.5z"
      fill="currentColor"
    />
  </svg>
</div>
`;

exports[`<IconCheck /> DOM element pass attributes defined in the "attrs" prop to the DOM element 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}"`;

exports[`<IconCheck /> DOM element render correctly when no props are passed 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c"
    height="1em"
    viewBox="0 0 725.7 725.7"
    width="1em"
  >
      
    <path
      d="M563.4 228.9L301.5 536.6l-139.2-153 45.6-41.7 75.4 82.8 7.9 8.7 9.2 10.1 8.9-10.4 7.6-8.9L516.6 189l46.8 39.9zm139.5 133.9c0 187.5-152.6 340.1-340.1 340.1S22.7 550.4 22.7 362.8c0-187.5 152.6-340.1 340.1-340.1 46 0 90.7 9.1 132.7 26.9 40.7 17.3 77.2 42 108.5 73.4 31.2 31.3 55.6 67.6 72.6 108 17.6 41.8 26.5 86.2 26.3 131.8zm-26.5.7c0-172.9-140.7-313.6-313.6-313.6S49.3 190.6 49.3 363.5 189.9 677 362.8 677c42.4 0 83.6-8.4 122.4-24.8 37.5-15.9 71.2-38.7 100-67.7 28.7-28.8 51.2-62.3 66.9-99.5 16.3-38.5 24.5-79.4 24.3-121.5z"
      fill="currentColor"
    />
  </svg>
</div>
`;

exports[`<IconCheck /> DOM element render correctly when no props are passed 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}"`;

exports[`<IconCheck /> styles responsively set misc styles on an icon 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c htz-d htz-e"
    height="1em"
    viewBox="0 0 725.7 725.7"
    width="1em"
  >
      
    <path
      d="M563.4 228.9L301.5 536.6l-139.2-153 45.6-41.7 75.4 82.8 7.9 8.7 9.2 10.1 8.9-10.4 7.6-8.9L516.6 189l46.8 39.9zm139.5 133.9c0 187.5-152.6 340.1-340.1 340.1S22.7 550.4 22.7 362.8c0-187.5 152.6-340.1 340.1-340.1 46 0 90.7 9.1 132.7 26.9 40.7 17.3 77.2 42 108.5 73.4 31.2 31.3 55.6 67.6 72.6 108 17.6 41.8 26.5 86.2 26.3 131.8zm-26.5.7c0-172.9-140.7-313.6-313.6-313.6S49.3 190.6 49.3 363.5 189.9 677 362.8 677c42.4 0 83.6-8.4 122.4-24.8 37.5-15.9 71.2-38.7 100-67.7 28.7-28.8 51.2-62.3 66.9-99.5 16.3-38.5 24.5-79.4 24.3-121.5z"
      fill="currentColor"
    />
  </svg>
</div>
`;

exports[`<IconCheck /> styles responsively set misc styles on an icon 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}@media (max-width: 79.9375em){.htz-d{position:relative}}@media (min-width: 80em){.htz-e{position:static}}"`;

exports[`<IconCheck /> styles set icon's color 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c"
    height="1em"
    viewBox="0 0 725.7 725.7"
    width="1em"
  >
      
    <path
      d="M563.4 228.9L301.5 536.6l-139.2-153 45.6-41.7 75.4 82.8 7.9 8.7 9.2 10.1 8.9-10.4 7.6-8.9L516.6 189l46.8 39.9zm139.5 133.9c0 187.5-152.6 340.1-340.1 340.1S22.7 550.4 22.7 362.8c0-187.5 152.6-340.1 340.1-340.1 46 0 90.7 9.1 132.7 26.9 40.7 17.3 77.2 42 108.5 73.4 31.2 31.3 55.6 67.6 72.6 108 17.6 41.8 26.5 86.2 26.3 131.8zm-26.5.7c0-172.9-140.7-313.6-313.6-313.6S49.3 190.6 49.3 363.5 189.9 677 362.8 677c42.4 0 83.6-8.4 122.4-24.8 37.5-15.9 71.2-38.7 100-67.7 28.7-28.8 51.2-62.3 66.9-99.5 16.3-38.5 24.5-79.4 24.3-121.5z"
      fill="currentColor"
    />
  </svg>
</div>
`;

exports[`<IconCheck /> styles set icon's color 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:#0B7EB5}.htz-c{fill:#FFF}"`;

exports[`<IconCheck /> styles set icon's color responsively 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c htz-d"
    height="1em"
    viewBox="0 0 725.7 725.7"
    width="1em"
  >
      
    <path
      d="M563.4 228.9L301.5 536.6l-139.2-153 45.6-41.7 75.4 82.8 7.9 8.7 9.2 10.1 8.9-10.4 7.6-8.9L516.6 189l46.8 39.9zm139.5 133.9c0 187.5-152.6 340.1-340.1 340.1S22.7 550.4 22.7 362.8c0-187.5 152.6-340.1 340.1-340.1 46 0 90.7 9.1 132.7 26.9 40.7 17.3 77.2 42 108.5 73.4 31.2 31.3 55.6 67.6 72.6 108 17.6 41.8 26.5 86.2 26.3 131.8zm-26.5.7c0-172.9-140.7-313.6-313.6-313.6S49.3 190.6 49.3 363.5 189.9 677 362.8 677c42.4 0 83.6-8.4 122.4-24.8 37.5-15.9 71.2-38.7 100-67.7 28.7-28.8 51.2-62.3 66.9-99.5 16.3-38.5 24.5-79.4 24.3-121.5z"
      fill="currentColor"
    />
  </svg>
</div>
`;

exports[`<IconCheck /> styles set icon's color responsively 2`] = `".htz-a{vertical-align:-0.15em}.htz-d{fill:#FFF}@media (max-width: 79.9375em){.htz-b{color:#0B7EB5}}@media (min-width: 80em){.htz-c{color:#2D2D2D}}"`;

exports[`<IconCheck /> styles set icon's fill 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c"
    height="1em"
    viewBox="0 0 725.7 725.7"
    width="1em"
  >
      
    <path
      d="M563.4 228.9L301.5 536.6l-139.2-153 45.6-41.7 75.4 82.8 7.9 8.7 9.2 10.1 8.9-10.4 7.6-8.9L516.6 189l46.8 39.9zm139.5 133.9c0 187.5-152.6 340.1-340.1 340.1S22.7 550.4 22.7 362.8c0-187.5 152.6-340.1 340.1-340.1 46 0 90.7 9.1 132.7 26.9 40.7 17.3 77.2 42 108.5 73.4 31.2 31.3 55.6 67.6 72.6 108 17.6 41.8 26.5 86.2 26.3 131.8zm-26.5.7c0-172.9-140.7-313.6-313.6-313.6S49.3 190.6 49.3 363.5 189.9 677 362.8 677c42.4 0 83.6-8.4 122.4-24.8 37.5-15.9 71.2-38.7 100-67.7 28.7-28.8 51.2-62.3 66.9-99.5 16.3-38.5 24.5-79.4 24.3-121.5z"
      fill="currentColor"
    />
  </svg>
</div>
`;

exports[`<IconCheck /> styles set icon's fill 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#0B7EB5}"`;

exports[`<IconCheck /> styles set icon's fill responsively 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c htz-d"
    height="1em"
    viewBox="0 0 725.7 725.7"
    width="1em"
  >
      
    <path
      d="M563.4 228.9L301.5 536.6l-139.2-153 45.6-41.7 75.4 82.8 7.9 8.7 9.2 10.1 8.9-10.4 7.6-8.9L516.6 189l46.8 39.9zm139.5 133.9c0 187.5-152.6 340.1-340.1 340.1S22.7 550.4 22.7 362.8c0-187.5 152.6-340.1 340.1-340.1 46 0 90.7 9.1 132.7 26.9 40.7 17.3 77.2 42 108.5 73.4 31.2 31.3 55.6 67.6 72.6 108 17.6 41.8 26.5 86.2 26.3 131.8zm-26.5.7c0-172.9-140.7-313.6-313.6-313.6S49.3 190.6 49.3 363.5 189.9 677 362.8 677c42.4 0 83.6-8.4 122.4-24.8 37.5-15.9 71.2-38.7 100-67.7 28.7-28.8 51.2-62.3 66.9-99.5 16.3-38.5 24.5-79.4 24.3-121.5z"
      fill="currentColor"
    />
  </svg>
</div>
`;

exports[`<IconCheck /> styles set icon's fill responsively 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}@media (max-width: 79.9375em){.htz-c{fill:#0B7EB5}}@media (min-width: 80em){.htz-d{fill:#2D2D2D}}"`;

exports[`<IconCheck /> styles set icon's size 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c htz-d"
    height="1em"
    viewBox="0 0 725.7 725.7"
    width="1em"
  >
      
    <path
      d="M563.4 228.9L301.5 536.6l-139.2-153 45.6-41.7 75.4 82.8 7.9 8.7 9.2 10.1 8.9-10.4 7.6-8.9L516.6 189l46.8 39.9zm139.5 133.9c0 187.5-152.6 340.1-340.1 340.1S22.7 550.4 22.7 362.8c0-187.5 152.6-340.1 340.1-340.1 46 0 90.7 9.1 132.7 26.9 40.7 17.3 77.2 42 108.5 73.4 31.2 31.3 55.6 67.6 72.6 108 17.6 41.8 26.5 86.2 26.3 131.8zm-26.5.7c0-172.9-140.7-313.6-313.6-313.6S49.3 190.6 49.3 363.5 189.9 677 362.8 677c42.4 0 83.6-8.4 122.4-24.8 37.5-15.9 71.2-38.7 100-67.7 28.7-28.8 51.2-62.3 66.9-99.5 16.3-38.5 24.5-79.4 24.3-121.5z"
      fill="currentColor"
    />
  </svg>
</div>
`;

exports[`<IconCheck /> styles set icon's size 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}.htz-d{font-size:6rem}"`;

exports[`<IconCheck /> styles set icon's size responsively 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c htz-d htz-e"
    height="1em"
    viewBox="0 0 725.7 725.7"
    width="1em"
  >
      
    <path
      d="M563.4 228.9L301.5 536.6l-139.2-153 45.6-41.7 75.4 82.8 7.9 8.7 9.2 10.1 8.9-10.4 7.6-8.9L516.6 189l46.8 39.9zm139.5 133.9c0 187.5-152.6 340.1-340.1 340.1S22.7 550.4 22.7 362.8c0-187.5 152.6-340.1 340.1-340.1 46 0 90.7 9.1 132.7 26.9 40.7 17.3 77.2 42 108.5 73.4 31.2 31.3 55.6 67.6 72.6 108 17.6 41.8 26.5 86.2 26.3 131.8zm-26.5.7c0-172.9-140.7-313.6-313.6-313.6S49.3 190.6 49.3 363.5 189.9 677 362.8 677c42.4 0 83.6-8.4 122.4-24.8 37.5-15.9 71.2-38.7 100-67.7 28.7-28.8 51.2-62.3 66.9-99.5 16.3-38.5 24.5-79.4 24.3-121.5z"
      fill="currentColor"
    />
  </svg>
</div>
`;

exports[`<IconCheck /> styles set icon's size responsively 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}@media (max-width: 79.9375em){.htz-d{font-size:6rem}}@media (min-width: 80em){.htz-e{font-size:7rem}}"`;

exports[`<IconCheck /> styles set misc styles on an icon 1`] = `
<div>
  <svg
    className="htz-a htz-b htz-c htz-d"
    height="1em"
    viewBox="0 0 725.7 725.7"
    width="1em"
  >
      
    <path
      d="M563.4 228.9L301.5 536.6l-139.2-153 45.6-41.7 75.4 82.8 7.9 8.7 9.2 10.1 8.9-10.4 7.6-8.9L516.6 189l46.8 39.9zm139.5 133.9c0 187.5-152.6 340.1-340.1 340.1S22.7 550.4 22.7 362.8c0-187.5 152.6-340.1 340.1-340.1 46 0 90.7 9.1 132.7 26.9 40.7 17.3 77.2 42 108.5 73.4 31.2 31.3 55.6 67.6 72.6 108 17.6 41.8 26.5 86.2 26.3 131.8zm-26.5.7c0-172.9-140.7-313.6-313.6-313.6S49.3 190.6 49.3 363.5 189.9 677 362.8 677c42.4 0 83.6-8.4 122.4-24.8 37.5-15.9 71.2-38.7 100-67.7 28.7-28.8 51.2-62.3 66.9-99.5 16.3-38.5 24.5-79.4 24.3-121.5z"
      fill="currentColor"
    />
  </svg>
</div>
`;

exports[`<IconCheck /> styles set misc styles on an icon 2`] = `".htz-a{vertical-align:-0.15em}.htz-b{color:inherit}.htz-c{fill:#FFF}.htz-d{position:relative}"`;
